pyproject.toml

```diff
@@ -37,43 +37,109 @@ disable_error_code = "attr-defined"
 warn_no_return = "False"
 
 # Changes mypy default to ignore all errors
+# TODO: the goal is for this to be empty
 [[tool.mypy.overrides]]
+# the list can be generated with:
+# mypy | tr ':' ' ' |  awk '{print $1}' | sort | uniq | sed 's/\.py//g' | sed 's|\/|\.|g' | xargs -I {} echo '"{}",'
 module = [
-    "pytorch_lightning.*",
+    "pytorch_lightning.accelerators.accelerator",
```

Comment:
Fun fact: There are 99 files ignored.

This means we are 99 PRs (or less) away from being fully typed